### PR TITLE
Fix windows issue on target name

### PR DESCRIPTION
**Pull request by jcarpent**

### CODE REVIEW

File: python.cmake [hunk 1]
```diff
@@ -394,6 +394,12 @@ MACRO(PYTHON_BUILD MODULE FILE)
   STRING(REGEX REPLACE "/|:" "_" FILE_TARGET_NAME
     "${CMAKE_CURRENT_BINARY_DIR}/${MODULE}/${FILE}c")
 
+  # Remove extra C:\* on Windows
+  IF(WIN32)
+    STRING(REGEX REPLACE "^[^:_]*:_" "" FILE_TARGET_NAME
+      "${FILE_TARGET_NAME}")
+  ENDIF(WIN32)
+
```
Comment (jmirabel) [inline python.cmake:402]: If this issue is too troublesome, we can use add_custom_command instead.

Comment (jcarpent) [reply python.cmake:402]: I agree. This would be the cleanest solution.

Comment (jcarpent) [reply python.cmake:402]: @nim65s What do you think?

Comment (nim65s) [reply python.cmake:402]: I don't have any useful comment for code enclosed in `IF(WIN32)`.

Comment (jcarpent) [reply python.cmake:402]: I was mentioning the suggestion made by @jmirabel to gather together all the generated Python files.

**Comment by jcarpent** [comment]
I've just updated the content of this PR and followed the recommandation made by @jmirabel which is very smooth.

File: python.cmake (hunk 1 above)
Comment (nim65s) [reply python.cmake]: Now I get it. Yes, that was clearly a good idea :)

**Pull request merged**